lib/src/test/java/android/support/graphics/drawable/VectorDrawableCompat.java

```diff
@@ -0,0 +1,44 @@
+package android.support.graphics.drawable;
+
+import android.graphics.Canvas;
+import android.graphics.ColorFilter;
+import android.graphics.PixelFormat;
+
+/**
+ *
+ */
+abstract class VectorDrawableCommon extends android.graphics.drawable.Drawable {
```

Comment:
Upgrading `support-v4` and `appcompat-v7` dependencies to version 25.1.0 and removing the `@aar` suffix makes it so copying this dummy class is not necessary to running the tests.